app/src/main/java/com/stl/skipthelibrary/Activities/ViewBookActivity.java

```diff
@@ -395,7 +397,9 @@ private void fillBookDescriptionFields(){
         title_element.setText(book.getDescription().getTitle());
         author_element.setText(book.getDescription().getAuthor());
         rating_element.setMax(book.getRating().getMaxRating());
-        rating_element.setNumStars((int) Math.round(book.getRating().getAverageRating()));
+        Log.d(TAG, "fillBookDescriptionFields: rating "+book.getRating());
+        rating_element.setRating((float)book.getRating().getAverageRating());
```

Comment:
```suggestion
rating_element.setNumStars(5);
        rating_element.setRating((float)book.getRating().getAverageRating());
```

Reply:
@gillianpierce so having this in the xml isn't equivalent to that¿ https://github.com/CMPUT301W19T09/SkipTheLibrary/pull/54/files#diff-3e5dab69d039442f636cec22cba9bc77R93

Reply:
oh yes I believe it is